README.md

```diff
@@ -1,110 +1,231 @@
-    <img src="https://gear-tech.io/images/logo-black.svg" width="240" alt="GEAR">
+    <img src="images/footer-logo.svg" width="400" alt="GEAR">
   </a>
 </p>
 
+<h1 align="center"> 
+The most advanced smart-contract engine
+</h1>
+
 <p align="center">
-  <b>Computational Component of Polkadot Network</b>
+ <img src="images/gear.gif" alt="Snow"><br>
+Gear is a Substrate-based smart-contract platform allowing anyone to run dApp in a few minutes.
 </p>
 
-<p align=center>
-    <a href="https://github.com/gear-tech/gear/actions/workflows/master.yml"><img src="https://github.com/gear-tech/gear/workflows/CI/badge.svg"></a>
-    <a href="https://github.com/gear-tech/gear/blob/master/LICENSE"><img src="https://img.shields.io/badge/License-GPL%203.0-success"></a>
+# 
+
+<div align="center">
+
+[![CI][c1]][c2]
+[![GitHubStars][g1]][g2] 
+[![Discord][d1]][d2] 
+[![Twitter][t1]][t2] 
+[![License][l1]][l2] 
+
+[c1]: https://github.com/gear-tech/gear/workflows/CI/badge.svg
+[c2]: https://github.com/gear-tech/gear/actions/workflows/CI.yaml
+
+[g1]: https://img.shields.io/github/stars/gear-tech/gear?style=flat-square&label=Stars
+[g2]: https://github.com/gear-tech/gear
+
+[t1]: https://img.shields.io/twitter/follow/gear_techs?style=social
+[t2]: https://twitter.com/gear_techs
+
+[d1]: https://img.shields.io/discord/891063355526217738?style=flat-square&label=Discord
+[d2]: https://discord.com/invite/7BQznC9uD9
+
+[l1]: https://img.shields.io/badge/License-GPL%203.0-success
+[l2]: https://github.com/gear-tech/gear/blob/master/LICENSE
+</div>
+
+<h4 align="center">Hit the <a href="https://github.com/gear-tech/gear">:star:</a> button to keep up with our daily progress!</h4>
+
+# ⚙️ What does Gear do?
+
+<p align="center">
+<img src="images/rust.png" height="64"><br>Gear provides the easiest and most cost-effective way <br>to run WebAssembly programs (smart-contracts) compiled from <br>many popular languages, such as Rust, C/C++ and more.
+</p>
+<p align="center">
+<img src="images/api.png" height="64"><br>Gear ensures very minimal, intuitive, and sufficient API <br>for running both newly written and existing programs <br>on multiple networks without the need to rewrite them.
+</p>
+<p align="center">
+<img src="images/state.png" height="64"><br>Smart Contracts are stored in the blockchain’s state <br>and are invoked preserving their state upon request.
+</p>
+<p align="center">
+<img src="images/apps.png" height="64"><br>GEAR enables a seamless transition to Web3, <br>enabling the running of dApps, microservices, middleware and open APIs.
 </p>
 
-<br/>
+### :fire: Key features
 
-Gear is a new Polkadot/Kusama parachain and most advanced L2 smart-contract engine allowing anyone to launch any dApp for networks with untrusted code.
+ - Programs run in WASM WM (near-native code execution speed)
+ - **Unique** :crown: : Parallelizable architecture (even greater speed)
+ - **Unique** :crown: : Actor model for message-passing communications - secure, effective, clear
```

Comment:
I'd prefer em-dash here instead of the hyphen. Here and below.
```suggestion
 - **Unique** :crown: : Actor model for message-passing communications — secure, effective, clear
```